File: csharp/Enqueue.cs
using System;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;

class EnqueueExample
{
    static async Task Main()
    {
        string apiKey = "YOUR_API_KEY";
        string endpoint = "https://app.pendless.com/api/enqueue/";

        var data = new
        {
            procedure = "enqueue",
            url = "https://example.com/login",
            prompt = "Open the login page, enter username and password",
            title = "Browser Automation: Login"
        };

        using (var client = new HttpClient())
        {
            client.DefaultRequestHeaders.Add("Authorization", $"Bearer {apiKey}");
            var content = new StringContent(System.Text.Json.JsonSerializer.Serialize(data), Encoding.UTF8, "application/json");

            var response = await client.PostAsync(endpoint, content);
            string respBody = await response.Content.ReadAsStringAsync();
            Console.WriteLine(respBody);
        }
    }
}

File: java/Enqueue.java
import java.io.*;
import java.net.HttpURLConnection;
import java.net.URL;

public class Enqueue {
    public static void main(String[] args) throws IOException {
        String apiKey = "YOUR_API_KEY";
        String endpoint = "https://app.pendless.com/api/enqueue/";

        String jsonInput = "{\"procedure\":\"enqueue\",\"url\":\"https://example.com/login\",\"prompt\":\"Open the login page, enter username and password\",\"title\":\"Browser Automation: Login\"}";

        URL url = new URL(endpoint);
        HttpURLConnection conn = (HttpURLConnection) url.openConnection();
        conn.setRequestMethod("POST");
        conn.setRequestProperty("Authorization", "Bearer " + apiKey);
        conn.setRequestProperty("Content-Type", "application/json");
        conn.setDoOutput(true);

        try (OutputStream os = conn.getOutputStream()) {
            byte[] input = jsonInput.getBytes("utf-8");
            os.write(input, 0, input.length);
        }

        try (BufferedReader br = new BufferedReader(
                new InputStreamReader(conn.getInputStream(), "utf-8"))) {
            StringBuilder response = new StringBuilder();
            String responseLine = null;
            while ((responseLine = br.readLine()) != null) {
                response.append(responseLine.trim());
            }
            System.out.println(response.toString());
        }
    }
}
